projects/cr-lib/src/lib/auth/confirm/confirm.page.ts

```diff
@@ -1,12 +1,11 @@
 /* tslint:disable:component-class-suffix */
 import { Component, OnInit } from '@angular/core';
 import {Member} from '../../api/member/member';
+import {ProfileService} from '../../api/profile/profile.service';
 import {RegStateService} from '../state/reg-state.service';
 import {Title} from '@angular/platform-browser';
 
 @Component({
-  // TODO: CI-11 update the references to this component.
-  // selector: 'confirm',
```

Comment:
I doubt the Confirm page will be accessed from a browser, so I'm pulling this oun until we find we need it.